path_to_dmat  : /home/mkz/catkin_ws/src/osm-localization/distance_maps/run1_.npy
#msc
# min_x : 751473.523660
# max_x : 777875.8610
# min_y : 3368749.3706
# max_y : 3392029.1009

# min_x : 754266.0
# max_x : 754366.0
# min_y : 3388880.94
# max_y : 3389280.94

# run 1
min_x : 719356.43731
max_x : 763470.4563
min_y : 3379403.5684
max_y : 3429942.0857
seed_x : 738912.92
seed_y : 3392799.59




#Runtime Parameters
map_resolution_x : 6.0
map_resolution_y : 6.87
num_particles : 1000
down_sample_size : 2.0
init_cov_linear : 50.0
init_cov_angular : 6.2831 
odom_cov_lin : 2.0
odom_cov_angular : 0.3
resampling_count : 8

use_pi_weighting : true
use_pi_resampling : true

road_width : 12
pi_gain : 1.0
queue_size : 50000
sync_queue_size : 10

project_cloud : false
